pallets/ajuna-nft-transfer/src/benchmarking.rs

```diff
@@ -64,7 +50,7 @@ fn create_service_account_and_prepare_avatar<T: Config>(
 
 fn enable_fee_payment<T: Config>(player: &T::AccountId) {
 	let prepare_fee = 100_000_000_000_000u128;
-	CurrencyOf::<T>::make_free_balance_be(player, prepare_fee.saturated_into());
+	T::BenchmarkHelper::set_account_balance(player, prepare_fee.saturated_into());
```

Comment:
Because of this:

```
error[E0277]: the trait bound `T: pallet_nfts::Config` is not satisfied
  --> pallets/ajuna-nft-transfer/src/benchmarking.rs:55:2
   |
55 |     CurrencyOf::<T>::make_free_balance_be(player, prepare_fee.saturated_into());
   |     ^^^^^^^^^^^^^^^ the trait `pallet_nfts::Config` is not implemented for `T`
   |
help: consider further restricting this bound
   |
53 | fn enable_fee_payment<T: Config + pallet_nfts::Config>(player: &T::AccountId) {
   |                                 +++++++++++++++++++++
```

Reply:
Ok, this is bad design by us. The pallet needs to transfer funds, but it doesn't expose what is used to manage funds. I prefer though to add the constraint `pallet_balances::Config` here. As this is the assumption that the nft-transfer pallet has currently, and it is fullfilled by our runtime.

In the future, we probably want to have a `set_balance` feature in our fee-handler, gated behind a runtime-benchmarks feature ;)

Reply:
Okay fixed it using `where_clause` in the benchmarks